src/lib/components/home/artwork_rails/__tests__/artwork_rail-tests.tsx

```diff
@@ -20,19 +19,23 @@ it("has a min height while data is loading", () => {
 
 it("has no min height when data is not loading", () => {
   const props = railProps(true) as any
-  const wrapper = shallow(<ArtworkRail {...props} />)
-  wrapper.setState({ didPerformFetch: true })
+  const rail = new ArtworkRail(props) as any
 
-  const instance = wrapper.instance() as ArtworkRail
-  const railStyle = instance.railStyle()
+  /**
+   * This is a less-than-ideal workaround because setState wasn't actually updating the state. Ideally we should bring
+   * back Enzyme or something similar once we figure out compatibility issues with RN 0.45+
+   */
+  rail.state = { didPerformFetch: true }
+
```

Comment:
Oh, this is probably due to `setState` being async, you need to use something like `setImmediate` to perform assertions on the next runloop tick.